diy_asylum_frontend/src/components/LandingPage/landing.css

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Start Bootstrap - Busines Frontpage (https://startbootstrap.com/template-overviews/business-frontpage)
+ * Copyright 2013-2017 Start Bootstrap
+ * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-business-frontpage/blob/master/LICENSE)
+ */
+
+body {
+  padding-top: 54px;
+}
+
+@media (min-width: 992px) {
+  body {
+    padding-top: 56px;
+  }
+}
+
+.business-header {
+  height: 50vh;
+  min-height: 300px;
+  background: url("../../img/landing_skygrass.png") center no-repeat scroll;
+  -webkit-background-size: auto;
```

Comment:
fun fact!
The create-react-app already comes with post-processing CSS prefixes. https://facebook.github.io/create-react-app/docs/post-processing-css

In other words, you don't have to write all of the "webkit" "moz" or "o" prefixes to background-size.
A single `background-size: auto;` will do the trick